rustic-babel.el

```diff
@@ -50,11 +50,14 @@
 
 (defvar rustic-babel-spinner nil)
 
-(defun rustic-babel-eval (dir)
+(defun rustic-babel-eval (dir toolchain-kw)
   "Start a rust babel compilation process in directory DIR."
   (let* ((err-buff (get-buffer-create rustic-babel-compilation-buffer-name))
          (default-directory dir)
-         (params '("cargo" "build" "--quiet"))
+         (toolchain (cond ((eq toolchain-kw 'nightly) "+nightly")
+                          ((eq toolchain-kw 'beta) "+beta")
+                          (t "+stable")))
```

Comment:
This only allows `nightly`/`beta`/`stable` channels by keyword.

Reply:
As opposed to?

Reply:
https://rust-lang.github.io/rustup/concepts/toolchains.html#toolchain-specification

Reply:
Ah I see the issue. We could expand the accepted format of `toolchain-kw` to allow for more free-form strings, and pass those as-is (or parse them?) down to the `cargo` line.

Reply:
Does https://github.com/brotzeit/rustic/commit/d71fbf5f3a39c3f126dc1ae3cafabd16c3160bc0 work for you ?

Reply:
That should do the trick.